include/bluetooth/bluetooth.h

```diff
@@ -29,6 +29,14 @@
 extern "C" {
 #endif
 
+/** @def BT_ID_DEFAULT
+ *
+ *  Convenience macro for specifying the default identity. This helps
+ *  make the code more readable, especially when only one identity is
+ *  supported.
+ */
+#define BT_ID_DEFAULT 0
```

Comment:
Is default identity always available or have to be generated using bt_id_create()? If it is always available, please state so in the comment.

Reply:
It can be created, if bt_id_create() is called before bt_enable(). Also, the settings storage will call bt_id_create() to create the default identity if one wasn't found in flash and the controller has no public address. I'll try to improve the documentation regarding this.